styles.css

```diff
@@ -272,6 +313,71 @@ li {
   background-color: var(--active-btn);
 }
 
+.displayed-project {
+  position: absolute;
+  top: 1.25rem;
+  left: 1rem;
+  width: 21rem;
+  height: 52rem;
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  text-align: left;
+  gap: 0.75rem;
+  background-color: white;
+  border-radius: 1rem;
+  z-index: 20;
+}
```

Comment:
- [ ] It is required to stick to the template design as much as possible. Your pop up box is not properly positioned to the center of the page as we have it on the guiding template you chose. Kindly look at the both designs and work your design alongside the Figma template you chose. This applies to both the desktop and mobile versions.

### Your Desing Desktop
![desktop](https://user-images.githubusercontent.com/26597432/215896762-3f59d26e-e160-42b1-b616-63bbd9b32f3f.png)

### Figma Design Desktop
![temp-desktop](https://user-images.githubusercontent.com/26597432/215896799-29684a96-4709-4ba0-957a-d4ecb4b88f40.png)


### Your Design Mobile
![mobile](https://user-images.githubusercontent.com/26597432/215896837-68123b74-f6cd-4679-ace9-a3440fccbdb1.png)

### Figma Design Mobile
![temp-mobile](https://user-images.githubusercontent.com/26597432/215896879-60a5b756-b5ce-46e9-afaf-8e34e69725fe.png)